src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -241,7 +241,7 @@ PARAM_DEFINE_FLOAT(MPC_XY_VEL_I, 0.02f);
 PARAM_DEFINE_FLOAT(MPC_XY_VEL_D, 0.01f);
 
 /**
- * Nominal horizontal velocity
+ * Nominal horizontal velocity in mission
```

Comment:
It's actually the maximal speed you can achieve in manual piloted flight.
Should it not be like MPC_VEL_MAN_MAX or similar?